src/backends/fixerio/index.js

```diff
@@ -145,8 +122,8 @@ class FixerIoBackend {
     return {
       source_ledger: params.source_ledger,
       destination_ledger: params.destination_ledger,
-      source_amount: sourceAmount,
-      destination_amount: destinationAmount
+      source_amount: sourceAmount.toString(),
+      destination_amount: destinationAmount.toString()
```

Comment:
@lumberj the reason I changed this to return strings was because it looks like the other backend is returning the amounts as strings, and I wanted consistency across all backends, so I could validate the response with the JSON schema.